### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by sexfreak** [bp2008/pingtracer#43]
would it be possible to ping 30 different hosts at once? 
I used smokeping in the past. this app likes nice. I would like to monitor more then 1 site at a time. 
thanks

**Comment by bp2008** [comment]
Yes.  The UI isn't designed to make it nice to input, but you should be able to enter 30 hosts into the host box, comma separated.

**Comment by sexfreak** [comment]
that was the trick it works great. 


    On Wednesday, September 27, 2023 at 07:36:20 PM EDT, bp2008 ***@***.***> wrote:  
 
 


Yes. The UI isn't designed to make it nice to input, but you should be able to enter 30 hosts into the host box, comma separated.

**Comment by HenryLoenwind** [comment]
And this is exactly what I needed. I've been looking for a tool to not only ping routers but also switches along the path for years!

The next thing I'm missing is selecting the source interface, or even better, comparatively ping from all of them.

**Comment by bp2008** [comment]
@HenryLoenwind 

I'm glad it is helpful.

.NET's Ping class does not apparently support choosing the source interface.  There is a [windows-native API](https://stackoverflow.com/questions/22575485/how-to-ping-using-a-specific-network-interface-or-a-specific-source-ip-address) that should work, but implementing that along with a network interface selector for each ping target would be way more work than I'm interested in doing in PingTracer 1.x.

I do have vague plans for a PingTracer v2 which would be much more scalable, where such a feature would be more at-home, but I never want to put in the weeks it would take to build it out from concept into a working app.

I would refer you to the commercial software that inspired PingTracer, which does have network interface selection according to this article: https://www.pingplotter.com/wisdom/article/nic-selection/
